buildspec_gnu.yml

```diff
@@ -96,6 +106,8 @@ phases:
       - if [ "$BUILD_STATUS" = "1" ] && [ "$CODEBUILD_BUILD_SUCCEEDING" = "0" ];
```

Comment:
CodeBuild defines:

`CODEBUILD_BUILD_SUCCEEDING: Whether the current build is succeeding. Set to 0 if the build is failing, or 1 if the build is succeeding.`

Suboptimal that it's opposite of what POSIX defines.